src/models/WeekDateObject.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * This ADO contains a grid of ADOs, representing a week.
+ * Which month a CDO "belongs" to depends on the majority
+ * of days in that week.
+ * The week starts on Gregorian Sunday
+ */
+
+import AtomicDateObject from "./AtomicDateObject";
+import DateTypeFormat from "../enums/DateTimeFormat";
+
+export default class WeekDateObject implements AtomicDateObject {
+  date: Date;
+  index: number;
+  next?: AtomicDateObject;
+  prev?: AtomicDateObject;
+  viewString: string;
+
+  split: boolean = false;
+  week: AtomicDateObject[];
+  innerHTML: string;
+  month: Date;
+
+  private dateTypeFormat: DateTypeFormat
+
+  static createInnerHTML = (week: AtomicDateObject[], split = false, month?: number): string => {
+    return week.reduce((acc: string, ado: AtomicDateObject, index: number) => {
+      if (split && month !== undefined) {
```

Comment:
No, because this is what renders the grouped version of the calendar, so it can't be default. (Joining is harder than splitting)
When this algo is called, it's because two week groups have been created and one is for the earlier month and the other is for the later month. 
I will add a comment so this is clearer to future generations!

Reply:
Sorry if I was unclear; I was asking about the conditional. It will evaluate true if either or both of the variables are falsy in ways other than `undefined`.

Reply:
if `split` is falsey won't it evaluate to false? They both have to be truthy for the condition to be true. Typescript will keep `month` from coming in as a non-number but not as undefined since it's optional. `split` is a boolean. false && true === false, right?

Reply:
You're right! I was reading that conditional entirely wrong.